### PR TITLE
(4/5) Multiple Issues: Fix intersections after mouse button release

### PR DESCRIPTION
**Pull request by andreasfelix**
**(4/5) Split up #177 in different PRs:**

Needs to be merged after #182

**Changes in this PR:**
- fix issue: for 3 and more monitors it was possible to create intersections between monitors (#35.2)
- check intersections after mouse button release: this makes it possible to place a monitor between to others

fix-intersections

### CODE REVIEW

File: src/Widgets/DisplaysOverlay.vala [hunk 1]
```diff
@@ -213,7 +213,11 @@ public class Display.DisplaysOverlay : Gtk.Overlay {
 
         display_widget.show_all ();
         display_widget.set_as_primary.connect (() => set_as_primary (display_widget.virtual_monitor));
-        display_widget.check_position.connect (() => check_intersects (display_widget));
+        display_widget.check_position.connect (() => {
+            check_intersects (display_widget);
+            close_gaps ();
+        });
```
Comment (jeremypw) [inline src/Widgets/DisplaysOverlay.vala]: Lint: Should be blank line after closure.

**Comment by andreasfelix** [comment]
> With three vertically stacked monitors, there is a small gap between the middle and bottom monitors
In 2ef6b0e I introduced a label with shows the exact position of the monitors.  Could you use this commit to check if there is a "real" gap or the gap is only optical?

**Comment by jeremypw** [comment]
@andreasfelix It looks like only a pixel gap on-screen so is probably some kind of rounding error - I'll investigate further.

**Comment by jeremypw** [comment]
Does not compile after merging master

**Comment by jeremypw** [comment]
As it looks like it is intended to drop a monitor between two others, my previous comment regarding moving the dropped monitor can be disregarded.

**Pull request merged**